contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -2981,3 +2981,24 @@ LEFT JOIN pg_foreign_server AS f ON u.srvid = f.oid
 LEFT JOIN pg_foreign_data_wrapper AS w ON f.srvfdw = w.oid
 WHERE w.fdwname = 'tds_fdw';
 GRANT SELECT ON sys.linked_logins TO PUBLIC;
+
+CREATE OR REPLACE VIEW sys.sql_expression_dependencies
+AS
+SELECT
+    CAST(0 as INT) AS referencing_id,
```

Comment:
Make sure that we have correct datatype for return values so that in future we can avoid rename during proper implementation

Reply:
Yes all return values datatypes are correct.